### PR TITLE
set rust-toolchain in bin to 1.70.0

### PR DESCRIPTION
**Pull request by Keksoj**
Cloning and compiling the sozu binary would fail because of the `rust-toolchain` file introduced in c60f5ee5. One dependency needs version 1.70.0 when doing `cargo install --path .` in the workspace `bin`:

```
error: failed to compile `sozu v0.15.3 (/home/emmanuel/sozu/bin)`, intermediate artifacts can be found at `/home/emmanuel/sozu/target`

Caused by:
  package `clap_lex v0.5.1` cannot be built because it requires rustc 1.70.0 or newer, while the currently active rustc version is 1.67.0
  Either upgrade to rustc 1.70.0 or newer, or use
  cargo update -p clap_lex@0.5.1 --precise ver
  where `ver` is the latest version of `clap_lex` supporting rustc 1.67.0
```

### CODE REVIEW

**Comment by FlorentinDUBOIS** [comment]
It miss the rust-version attribute in the cargo.toml file.

**Review by FlorentinDUBOIS — CHANGES_REQUESTED** [review]
You need to update the rust-version in the Cargo.toml file as well.

**Comment by FlorentinDUBOIS** [comment]
Do you have check for other crate as well ?

File: bin/rust-toolchain
Comment (FlorentinDUBOIS) [inline bin/rust-toolchain:1]: Could you move this file at the root of the repository and update every `Cargo.toml`'s field `rust-version`?
We could remove the other `rust-toolchain` if the override is working properly on sub-folder.

**Review by FlorentinDUBOIS — CHANGES_REQUESTED** [review]

File: bin/rust-toolchain
Comment (Keksoj) [reply bin/rust-toolchain:1]: done!

**Pull request merged**